static/elements/chromedash-stack-rank.js

```diff
@@ -93,94 +157,141 @@ class ChromedashStackRank extends LitElement {
-        el.scrollIntoView(true, {behavior: 'smooth'});
-      }
+  scrollToPosition(e) {
+    let hash;
+    if (e) {
+      hash = e.currentTarget.getAttribute('href');
+    } else if (location.hash) {
+      hash = decodeURIComponent(location.hash);
+    }
+
+    if (hash) {
+      const el = this.shadowRoot.querySelector('.stack-rank-list ' + hash);
+      el.scrollIntoView(true, {behavior: 'smooth'});
     }
   }
 
   sort(e) {
     e.preventDefault();
 
-    const order = e.currentTarget.dataset.order;
-    sortBy_(order, this.viewList);
-    switch (order) {
-      case 'percentage':
-        this.sortOrders.percentage.activated = true;
-        this.sortOrders.percentage.reverse = !this.sortOrders.percentage.reverse;
-        break;
-      case 'property_name':
-        this.sortOrders.property_name.activated = true;
-        this.sortOrders.property_name.reverse = !this.sortOrders.property_name.reverse;
-        break;
-    }
+    const parts = e.target.dataset.order.split('-');
+    this.sortType = parts[0];
+    this.sortReverse = Boolean(parts[1]);
 
-    this._updateSortIcons();
-    if (this.sortOrders[order].reverse) {
-      this.viewList.reverse();
-    }
+    const newViewList = [...this.viewList];
+    this.viewList = sortBy_(this.sortType, this.sortReverse, newViewList);
+  }
+
+  renderSubHeader() {
+    return html`
+      <div id="subheader">
+        <p class="title-text">Showing <span>${this.viewList.length}</span> properties</p>
+        <div id="dropdown-selection">
+          <sl-dropdown>
+            <sl-button slot="trigger" variant="text">
+              <iron-icon icon="chromestatus:sort"></iron-icon>
+              SORT BY
+            </sl-button>
+            <sl-menu @click="${this.sort}">
+              <sl-menu-item ?checked=${this.sortType == 'percentage' && !this.sortReverse}
+                data-order="percentage">
+                Most used
+              </sl-menu-item>
+              <sl-menu-item ?checked=${this.sortType == 'percentage' && this.sortReverse}
+                data-order="percentage-reverse">
+                Least used
+              </sl-menu-item>
+              <sl-menu-item ?checked=${this.sortType == 'property_name' && this.sortReverse}
+                data-order="property_name-reverse">
+                Name (A-Z)
+              </sl-menu-item>
+              <sl-menu-item ?checked=${this.sortType == 'property_name' && !this.sortReverse}
+                data-order="property_name">
+                Name (Z-A)
+              </sl-menu-item>
+            </sl-menu>
+          </sl-dropdown>
+        </div>
+      </div>
+    `;
   }
 
-  _updateSortIcons() {
-    this.propertyNameSortIcon = this._getOrderIcon('property_name');
-    this.percentSortIcon = this._getOrderIcon('percentage');
+  renderStackRank() {
+    return html`
+      ${this.viewList.map((item) => html`
+        <li class="stack-rank-item" id="${item.property_name}" tabindex="0">
```

Comment:
Do we need `tabindex="0"`?  I think that makes the row selectable by using the tab key, but it is not a link or anything, so I don't see the purpose of selecting the row.  If someone was trying to navigate using tab and enter keys, having all these extra tab-able elements would mean that they need to press tab 1/3 more often to get to what they want.

Reply:
I see. It is removed!